src/abstract.js

```diff
@@ -828,20 +828,24 @@ Sk.abstr.gattr = function (obj, nameJS, canSuspend) {
         ret = Sk.misceval.callsimOrSuspend(f, new Sk.builtin.str(nameJS));
     }
 
-    ret = Sk.misceval.chain(ret, function(ret) {
+    var eventualresult = Sk.misceval.chain(undefined, function () {
+        return obj;
```

Comment:
@meredydd I've also changed this because the obj is a suspension aswell. But it may be that I should have done it differently, because it sortof feels like the same kind of issue as I mentioned in the PR.